### PR TITLE
Workaround for v8 tick processor bug

### PR DESCRIPTION
**Pull request by mafintosh**
Should fix https://github.com/davidmarkclements/0x/issues/192 until we can fix it in V8

### CODE REVIEW

File: lib/v8-log-to-ticks.js [hunk 1]
```diff
@@ -15,7 +15,28 @@ module.exports = wrapper
...
+  data = data.split('\n').map(escapePath).join('\n')
+  await writeFile(dest, data)
+  return dest
+}
+
+function escapePath (line) {
+  const startPath = line.indexOf(':\\')
+  if (startPath === -1) return line
+  // \u005c is the unicode escape for the \ char
+  return line.slice(0, startPath) + line.slice(startPath).replace(/\\/g, '\\u005c')
```
Comment (mcollina) [inline lib/v8-log-to-ticks.js]: If the V8 fix is in, would this code break?

**Comment by AlanSl** [comment]
I've tested this on Windows 10 and it appears to fix https://github.com/davidmarkclements/0x/issues/192 so LGTM functionality-wise

File: lib/v8-log-to-ticks.js (hunk 1 above)
Comment (mafintosh) [reply lib/v8-log-to-ticks.js]: Potentially yes, depending on how the fix lands (my current fork just disables all unescaping in their parser, which *would* break this).

Once that lands we have time to fix it here tho, so I think this is okay for now.

Comment (mafintosh) [reply lib/v8-log-to-ticks.js]: Actually only replacing [^\]\ with the escape sequence properly future proofs a lot better :+1:

**Comment by AlanSl** [comment]
One observed issue which is maybe something for a separate PR - the names are coming out fine, but 0x is missclassifying some `app` and `deps` frames. For example from `0x/examples/rest-api` on Windows:

image

I'm going to do some more tests to see if it always misclassifies or if it's to do with particulars here such as the app code being in a subfolder and the dependency being installed outside of the project folder

**Comment by AlanSl** [comment]
Probably. Let's release this and I'll post it as a separate issue.

LGTM

**Pull request merged**